sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/ier12-integration.scheduler.ts

```diff
@@ -19,53 +22,36 @@ export class IER12IntegrationScheduler extends BaseScheduler<GeneratedDateQueueI
     super(schedulerQueue, queueService);
   }
 
-  /**
-   * To be removed once the method {@link process} is implemented.
-   * This method "hides" the {@link Process} decorator from the base class.
-   */
-  async processQueue(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
-  /**
-   * When implemented in a derived class, process the queue job.
-   * To be implemented.
-   */
-  protected async process(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
   /**
    * Identifies all the applications which are in assessment
    * for a particular institution and generate the request file.
    * @params job has generatedDate Date in which the assessment for
    * particular institution is generated.
-   * @returns Processing result log.
+   * @returns processing result log.
    */
-  @Process()
-  async processIER12File(
-    job: Job<GeneratedDateQueueInDTO | undefined>,
-  ): Promise<QueueProcessSummaryResult[]> {
-    this.logger.log(
-      `Processing IER integration job ${job.id} of type ${job.name}.`,
-    );
-    this.logger.log("Executing IER 12 file generation ...");
+  protected async process(
+    job: Job<GeneratedDateQueueInDTO>,
+    processSummary: ProcessSummary,
```

Comment:
Please add the comment for processSummary.

Reply:
Added.